docs/source/admin/upgrading.rst

```diff
@@ -0,0 +1,159 @@
+.. _admin/upgrading:
+
+====================
+Upgrading JupyterHub
+====================
+
+JupyterHub offers easy upgrade pathways between minor versions. This
+document describes how to do these upgrades. 
+
+If you are using a JupyterHub distribution, you should consult the
+distribution's documentation on how to upgrade. This document is if
+you have set up your own JupyterHub without using a distribution.
+
+It is long because is pretty detailed! Most likely, upgrading 
+JupyterHub is painless, quick and with minimal user interruption.
+
+Read the Changelog
+==================
+
+The `changelog <changelog.html>`_ contains information on what has
+changed with the new JupyterHub release, and any deprecation warnings.
+Read these notes to familiarize yourself with the coming changes. There
+might be new releases of authenticators & spawners you are using, so
+read the changelogs for those too!
+
+Notify your users
+=================
+
+If you are using the default configuration where ``configurable-http-proxy``
+is managed by JupyterHub, your users will see service disruption during
+the upgrade process. You should notify them, and pick a time to do the
+upgrade where they will be least disrupted.
+
+If you are using a different proxy, or running ``configurable-http-proxy``
+independent of JupyterHub, your users will be able to continue using notebook
+servers they had already launched, but will not be able to launch new servers
+nor sign in. This gives you more freedom to do upgrades without affecting
+users *too much*.
```

Comment:
I think the "too much" doesn't help the reader. What is "too much"?

Maybe replace the sentence with "During the upgrade users won't be able to login and start new session, existing notebook sessions will continue uninterrupted."?

Reply:
Or link to the section on distributions in this documentation